themes/bootstrap3/templates/Recommend/SideFacets/cluster-list.phtml

```diff
@@ -20,6 +20,13 @@
   <?php foreach ($this->cluster['list'] as $i => $thisFacet): ?>
     <?php if ($i == $this->facets_before_more): ?>
       <?php
+        $queryParams = [
+            'facet' => $this->title,
+            'facetop' => $thisFacet['operator'],
+            'facetexclude' => $this->allowExclude ? 1 : 0,
+            'searchClassId' => $this->searchClassId
+        ];
+        $facetLightboxParams = http_build_query($queryParams, '', '&amp;');
```

Comment:
This seems like a partially implemented change. Do we need to merge #2362 and update this before it is ready to merge?

Reply:
...and #2362 has now been merged, so you should be able to merge dev-9.0 back into this and resolve the issue.

Reply:
This was accidentally included here, not related to the source id change.